dynamic_instrumentation/src/point/cast.rs

```diff
@@ -10,7 +10,7 @@ use rustc_span::DUMMY_SP;
 
 use crate::{
     arg::{ArgKind, InstrumentationArg},
-    deref::remove_outer_deref,
```

Comment:
would like a similar re-export here to have `mir_utils::remove_outer_deref`

Reply:
Fixed in https://github.com/immunant/c2rust/pull/559/commits/50d305f1f2510161d8326d472dd561c0d87fa9e6.